snowfakery_edu_example.recipe.yml

```diff
@@ -7,9 +7,15 @@
 
 - plugin: faker_edu.Provider
 - object: Account
+  nickname: college
```

Comment:
Is there a reason you are suggesting using nickname here instead of a comment to indicate which is which?

Reply:
not really? Mostly since I've ended up using nicknames for other pieces of recipes, so I figured may as well create them...